face_track/main.py

```diff
@@ -27,9 +27,9 @@
 
 print "Start face-tracking node"
 ft = FaceTrack()
-
+ft.do_look_at_actions()
 while not rospy.is_shutdown():
-	ft.do_look_at_actions()
+	#ft.do_look_at_actions()
```

Comment:
Hi Linas,
 I could replace it with rospy.spin(), it would do something similar, and would be almost equivalent to what i did. There is a time.sleep in while loop which makes the loop delay by 0.1 seconds. This in my opinion is not wrong. Also I meant to remove ft.do_look_at_actions from the loop for several reasons, I will put them here in comments below